atomics/include/desul/atomics/HIP.hpp

```diff
@@ -261,6 +261,77 @@ __device__ inline
     atomic_fetch_or(T* dest, T val, MemoryOrder, MemoryScopeCore) {
   return atomic_fetch_or(dest, val, MemoryOrder(), MemoryScopeDevice());
 }
+
+}
+
+#define DESUL_HIP_GCC_INTEGRAL_OP_ATOMICS_COMPATIBILITY(MEMORY_ORDER, MEMORY_SCOPE)                 \
```

Comment:
This addition is not captured in the description of the PR

Reply:
added to commit message